src/__tests__/App.test.tsx

```diff
@@ -1,15 +1,20 @@
 import userEvent from '@testing-library/user-event';
+import { act } from 'react-dom/test-utils';
 
 import App from '../components/App';
 import { render } from './utils/setupTests';
 import { screen } from './utils/test-utils';
 
 describe('App', () => {
-  test('Should render search view on default route', () => {
+  test('Should render search view on default route', async () => {
     render(<App />);
 
+    await act(async () => {
+      jest.runOnlyPendingTimers();
+    });
+
```

Comment:
some render statements were causing 'not wrapped in act' errors due to state updates occurring after the test finished. adding runOnlyPending timers within an act call resolved these errors.

```
console.error node_modules/react-dom/cjs/react-dom.development.js:530
  Warning: An update to <component> inside a test was not wrapped in act(...).

  When testing, code that causes React state updates should be wrapped into act(...):

  act(() => {
    /* fire events that update state */
  });
  /* assert on the output */

  This ensures that you're testing the behavior the user would see in the browser. Learn more at https://fb.me/react-wrap-tests-with-act
      in <component>
      ```